test/integration/acceptance/tx_acceptance_test.cpp

```diff
@@ -0,0 +1,393 @@
+ * @when sending transactions with an empty signedBlob
+ * @then receive STATELESS_VALIDATION_FAILED status
+ */
+TEST(AcceptanceTest, DISABLED_TransactionEmptySignedblob) {
+  shared_model::proto::Transaction tx =
+      TestTransactionBuilder()
+          .txCounter(2)
+          .createdTime(iroha::time::now())
+          .creatorAccountId(kAdmin)
+          .addAssetQuantity(kAdmin, kAsset, "1.0")
+          .build();
+  iroha::protocol::Signature protosig;
+  protosig.set_pubkey(
+      shared_model::crypto::toBinaryString(kAdminKeypair.publicKey()));
+  protosig.set_signature(
+      shared_model::crypto::toBinaryString(shared_model::crypto::Blob("")));
+  auto *s1 = new shared_model::proto::Signature(protosig);
+  tx.addSignature(
+      shared_model::detail::PolymorphicWrapper<shared_model::proto::Signature>(
+          s1));
+  integration_framework::IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(tx, checkStatelessInvalid)
+      .done();
+}
+/**
+ * @given some user
+ * @when sending transactions with Invalid PublicKey
+ * @then receive STATELESS_VALIDATION_FAILED status
+ */
+TEST(AcceptanceTest, DISABLED_TransactionInvalidPublicKey) {
+  shared_model::proto::Transaction tx =
+      TestTransactionBuilder()
+          .txCounter(2)
+          .createdTime(iroha::time::now())
+          .creatorAccountId(kAdmin)
+          .addAssetQuantity(kAdmin, kAsset, "1.0")
+          .build();
+  auto signedBlob = shared_model::crypto::CryptoSigner<>::sign(
+      shared_model::crypto::Blob(tx.payload()), kAdminKeypair);
+  iroha::protocol::Signature protosig;
+  protosig.set_pubkey(shared_model::crypto::toBinaryString(
+      shared_model::crypto::PublicKey(std::string(32, 'a'))));
+  protosig.set_signature(shared_model::crypto::toBinaryString(signedBlob));
+  auto *s1 = new shared_model::proto::Signature(protosig);
+  tx.addSignature(
+      shared_model::detail::PolymorphicWrapper<shared_model::proto::Signature>(
+          s1));
+  integration_framework::IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(tx, checkStatelessInvalid)
+      .done();
+}
+/**
+ * @given some user
+ * @when sending transactions with Invalid SignedBlock
+ * @then receive STATELESS_VALIDATION_FAILED status
+ */
+TEST(AcceptanceTest, DISABLED_TransactionInvalidSignedBlob) {
+  shared_model::proto::Transaction tx =
+      TestTransactionBuilder()
+          .txCounter(2)
+          .createdTime(iroha::time::now())
+          .creatorAccountId(kAdmin)
+          .addAssetQuantity(kAdmin, kAsset, "1.0")
+          .build();
+  auto signedBlob = shared_model::crypto::CryptoSigner<>::sign(
+      shared_model::crypto::Blob(tx.payload()), kAdminKeypair);
+  iroha::protocol::Signature protosig;
+  protosig.set_pubkey(
+      shared_model::crypto::toBinaryString(kAdminKeypair.publicKey()));
+  auto raw = signedBlob.blob();
+  raw[0] = (raw[0] == std::numeric_limits<uint8_t>::max() ? raw[0] + 1 : 0);
```

Comment:
Looks strange. Actually it is equal to `raw[0] = 0`. What if raw[0] == 0? it will stay 0 and block will not be spoiled. Can you explain what does this line mean?